core/jobs/src/test/java/com/cognifide/aet/job/common/comparators/source/SourceComparatorRegExpTest.java

```diff
@@ -53,7 +53,7 @@ public void setUp() {
   }
 
   @Theory
-  public void removeEmptyLines(String[] dataPoint) throws Exception {
+  public void removeEmptyLines(String[] dataPoint) {
     String input = dataPoint[0];
```

Comment:
Could We also add some tests for `MARKUP`, `CONTENT` and `ALLFORMATTED` types?
I woinder if we could have one input HTML file and three expected output files that shows the difference in code formatting..